### ISSUE TITLE
Design multiple sequences for the same input structure

### ISSUE THREAD

**Issue by prateekbansal97** [generatebio/chroma#48]
Hello, 

Thanks for Chroma! It is a pretty great tool. 

I do have a query - is there a way to design multiple sequences (say 10) for the same input structure? I am new to Chroma and have tried running the design code multiple times with the same structure, but it gives out the same output. 

Any help in this matter would be appreciated. Thank you!

Here is the code I am using (same as the one in the README)
```
protein = Protein(f'/path/to/protein.pdb',device='cuda')
protein = chroma.design(protein)
protein.to(f"output_path.pdb")
```

**Comment by wujiewang** [comment]
Hey, thanks for the question.

I think you can do this by parsing a batch of backbones by calling

`X_designed, C, S_designed = chroma.design_network.sample(X.cuda(), C.cuda(),  sampling_method="potts")`

where X has shpe (Nbatch x Nres x 4 x 3) and C has shape (Nbatch x Nres)

**Comment by prateekbansal97** [comment]
Hi @wujiewang ,

Thanks for the reply. 

In this case, I generated X and C using `X, C, S = protein.to_XCS()` but got an `X.shape =  [1, Nres, 4, 3]`. How do I make it into size of the batch?

**Comment by wujiewang** [comment]
It is simpler than you think, you just need to copy X and C several time and concatenate. something like : 

`torch.cat( [X] * 10, dim=0 )`

**Comment by prateekbansal97** [comment]
The solution you gave is working. Thank you so much!
